models/DataObject/Data/InputQuantityValue.php

```diff
@@ -50,6 +50,6 @@ public function __toString(): string
             $value .= ' ' . $translator->trans($this->getUnit()->getAbbreviation(), [], 'admin');
         }
 
-        return $value;
+        return $value ? (string)$value : '';
```

Comment:
getValue can only return string or null. So this should be enough:
```suggestion
        return $value ?? '';
```

Reply:
had it in my offline repo like this, don't know why I changed it in the fork ;-)